src/Functions.Worker.Extensions.OpenAI/OpenAIModels.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.OpenAI;
+
+static class OpenAIModels
+{
+    // Reference - https://platform.openai.com/docs/models
+
+    /// <summary>
+    /// GPT 3 Turbo, refer to https://platform.openai.com/docs/models/continuous-model-upgrades for exact model being pointed to
+    /// </summary>
+    internal const string Gpt_35_Turbo = "gpt-3.5-turbo";
+
+    /// <summary>
+    /// The default embeddings model, currently pointing to text-embedding-3-small
+    /// </summary>
+    internal const string DefaultEmbeddingsModel = "text-embedding-3-small";
```

Comment:
Let's add a warning comment here that changing the default embeddings model is a breaking change.

The reason it's a breaking change is because customers will index their data using a particular embeddings model, and will then store those embeddings in a vector database for future lookups. Changing this default model will cause those lookups to start misbehaving if they don't match the data that was previously ingested into the vector DB.